SalesforceSDK/Salesforce.Universal.Template/Salesforce.Universal.Template.Shared/Pages/MainPage.xaml.cs

```diff
@@ -53,11 +57,41 @@ namespace $safeprojectname$.Pages
     /// </summary>
     public sealed partial class MainPage : NativeMainPage
     {
+        private const string ApiVersion = "v31.0";
```

Comment:
I'd recommend creating an ```ApiUtils``` class in the Mobile SDK library itself to store this constant. Makes updates to API version easier to do, as opposed to having to change it in each sample app.